cli/client.go

```diff
@@ -406,22 +406,7 @@ func VersionAction(c *cli.Context) error {
 
 var defaultBaseURL = "https://app.viam.com:443"
 
-func checkBaseURL(c *cli.Context, auth *config) (*url.URL, []rpc.DialOption, error) {
-	// If base URL was not specified, assume cached base URL. If no base URL is
-	// cached, assume default base URL.
-	baseURL := c.String(baseURLFlag)
-	if baseURL == "" {
-		baseURL = defaultBaseURL
-		if auth.Auth != nil && auth.Auth.BaseURL != "" {
-			baseURL = auth.Auth.BaseURL
-		}
-	}
-
-	if auth.Auth != nil && auth.Auth.BaseURL != baseURL {
-		return nil, nil, fmt.Errorf("cached base URL for this session is %q, "+
-			"please logout and login again to use provided base URL %q", auth.Auth.BaseURL, baseURL)
-	}
-
+func parseBaseURL(baseURL string, verifyConnection bool) (*url.URL, []rpc.DialOption, error) {
```

Comment:
@benjirewis are you okay with this function signature change? 

I wanted to add some basic tests for this function and prior signature was not as simple to test (and it was hard to realize that it was going to use the baseURLFlag inside a checking function)

Happy to revert and find some other way to do this if you like the way you did it.

Reply:
No this looks good to me! Thanks for adding the extra param around connection verifying.